src/components/SkillsSection/SkillsSection.js

```diff
@@ -152,6 +156,11 @@ const styles = ({breakpoints, palette, spacing}) => ({
     marginBottom: spacing(2),
     borderBottom: 'solid #e0e0e0 1px',
   },
+  displayNone: {
+    [breakpoints.down('xs')]: {
+      display: 'none',
+    },
+  },
```

Comment:
Why is this `xs` while the one below is `sm`? Also the name here could be better, probably something like "hideOnMobile"?

Reply:
I removed this part since it's a children of the container above, so it should be disappear when the parent display = none.